src/main/java/com/softserve/academy/spaced/repetition/controller/AuditController.java

```diff
@@ -28,11 +30,12 @@
      * @return - list of audit
      */
     @GetMapping("api/admin/audit")
-    public ResponseEntity<List<AuditPublicDTO>> getFullAuditList() {
-        List<Audit> auditList = auditService.getFullAuditList();
-        Link link = linkTo(methodOn(AuditController.class).getFullAuditList()).withSelfRel();
-        List<AuditPublicDTO> auditDTOList = DTOBuilder.buildDtoListForCollection(auditList, AuditPublicDTO.class, link);
-        return new ResponseEntity<>(auditDTOList, HttpStatus.OK);
+    public ResponseEntity<Page<AuditPublicDTO>> getFullAuditList(@RequestParam(name = "p", defaultValue = "1") int pageNumber, @RequestParam(name = "sortBy") String sortBy, @RequestParam(name = "asc") boolean ascending) {
```

Comment:
All @ RequestParam are required? The same question for other controllers.

Reply:
I think it is required